src/field_theory/polynomial_galois_group.lean

```diff
@@ -324,6 +332,86 @@ begin
   rw [aeval_def, map_root_of_splits _ (splitting_field.splits p) hp],
 end
 
+instance {p : polynomial ℚ} : fact (p.splits (algebra_map ℚ ℂ)) :=
+⟨is_alg_closed.splits_codomain p⟩
+
+lemma gal_action_hom_bijective_of_prime_degree_aux {p : polynomial ℚ} :
```

Comment:
Could you add a docstring to this lemma and the next one, and mention the next lemma in the file-level docstring?